rails-commenting.rb

```diff
@@ -56,8 +56,8 @@ def blog_post_params
 
 # app/models/blog_post.rb
 
-# 9)
+# 9) This is a class to give the spefoco attributes to BlogPost objects
```

Comment:
Absolutely, I would maybe mention again how this is inheriting from ApplicationRecord